ci/requirements.txt

```diff
@@ -1,7 +1,7 @@
 black
 dask
-intake
-matplotlib
+intake-xarray
+matplotlib>=2.0.2,<3.0
```

Comment:
can't restrict to matplotlib 2.X

Reply:
Agree - we need to fix the vertical split scale issue.